lang/java/tools/src/test/compiler/output-string/avro/examples/baseball/Player.java

```diff
@@ -117,8 +117,8 @@ public Player(java.lang.Integer number, java.lang.String first_name, java.lang.S
   public void put(int field$, java.lang.Object value$) {
     switch (field$) {
     case 0: number = (java.lang.Integer)value$; break;
-    case 1: first_name = (java.lang.String)value$; break;
-    case 2: last_name = (java.lang.String)value$; break;
+    case 1: first_name = value$!=null?value$.toString():null; break;
+    case 2: last_name = value$!=null?value$.toString():null; break;
```

Comment:
```suggestion
    case 2: last_name = value$ != null ? value$.toString(): null; break;
```

Reply:
Thanks for picking this up so quickly, I think you've missed this one @mhelleborg